# This contains the position of a point in free space
float64 xThomas
float64 yThomas
float64 xLightning
float64 yLightning
